webapp/sdf/src/components/item-form-component/item-form-component.stories.js

```diff
@@ -0,0 +1,18 @@
+import ItemFormComponent from "./item-form-component.vue";
+
+export default {
+    title: 'ItemFormComponent',
+    component: ItemFormComponent,
+    argTypes: {
+      onSubmit: { action: "Create Item"},
+    },
+  };
+
+  const Template = (args, { argTypes }) => ({
+    props: Object.keys(argTypes),
+    components: { ItemFormComponent },
+    template: '<ItemFormComponent @submitEvent="onSubmit"/>',
+    
+});
+  
+  export const Base = Template.bind({});
```

Comment:
Just clean code improvements.
```suggestion
  export const Base = Template.bind({});